spacelift/internal/structs/user_group.go

```diff
@@ -0,0 +1,12 @@
+package structs
+
+type SpaceAccessRule struct {
```

Comment:
In a similar spirit, this could be named `Role`

Reply:
The concept of a Role within user management already exists.
The role can be admin, owner, and user, so this name is not available.

Reply:
similar to the other one, I'll stick with what's in the api definition